data/human/news.txt

```diff
@@ -1479,6 +1571,10 @@ news "tourist"
 			"Can I speak to the manager? There was a scratch on my cabin-door and it really irritated me. I think my whole vacation is ruined now."
 			"Can I speak to the manager? The landing on this planet was a bit rough and I'm seriously considering filing a complaint."
 			"Warn me if you see my partner or my secret lover. They learned about each other and now they're both trying to kill me."
+			"Get away from me! I already told twenty of your colleagues that I don't need a taxi service."
+			"You there, where are my bags! What do you mean by 'you don't know'? You look like one of those dockworkers."
```

Comment:
Can this be changed to using double quotes for 'you don't know'?

Reply:
No, because there are already double-quotes around this whole section. (It would look strange to use double quotes here too.)